src/ESPUI.cpp

```diff
@@ -1031,7 +1031,26 @@ void ESPUIClass::updateTime(uint16_t id, int clientId)
     updateControl(id, clientId);
 }
 
-void ESPUIClass::clearGraph(uint16_t id, int clientId) { }
+void ESPUIClass::clearGraph(uint16_t id, int clientId) { 
+    do // once
+    {
+        Control* control = getControl(id);
+        if (!control)
+        {
+            break;
+        }
+
+        DynamicJsonDocument document(jsonUpdateDocumentSize);
+        JsonObject root = document.to<JsonObject>();
+
+        root[F("type")] = 109;
```

Comment:
I think you can still use constants here
Graph + UpdateOffset

Reply:
Done in the latest commit :)